client/me/purchases/remove-purchase/index.jsx

```diff
@@ -176,28 +355,273 @@ const RemovePurchase = React.createClass( {
 							comment: "'it' refers to a product purchased by a user"
 						} )
 					}
+
 				</p>
 
 				{ includedDomainText }
 			</div>
 		);
 	},
 
+	renderQuestionOne() {
+		const reasons = {},
+			ordered_reasons = [];
+
+		const could_not_install_input = (
```

Comment:
Is there a reason you're using snake_case here? We use camelCase whenever possible.